spec/rubocop/cop/solidus/reimbursement_hook_deprecated_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Solidus::ReimbursementHookDeprecated, :config do
+  let(:config) { RuboCop::Config.new('Solidus/ReimbursementHookDeprecated' => { 'Enabled' => true }) }
+
+  it 'registers an offense when using `#reimbursement_success_hooks`' do
+    expect_offense(<<~RUBY)
+      reimbursement_success_hooks
```

Comment:
I'm not sure but do these statements come in any other form in the code ?
Or
do they appear like a single statement without arguments only like we have in the spec file here.

It might be worth investigating a little more if there is any other way this statement can be present in the code like with arguments or other options maybe.

If we find there is no other way for this statement to appear then we can go ahead.